### DISCUSSION TITLE
How can I use a binding in the SQL or DynamoDB Query Transformer?

### DISCUSSION THREAD

**Discussion by lollococce** [Budibase/budibase#5046] [Help]
Hi!

I might need some binding input in the transformer of a DynamoDB query. Is there anyway specifically to call the binding var in the JS script?

Lorenzo

**Comment by melohagan** [comment]
Hi @lollococce 

At the moment there is no way to include the binding vars in the transformers. 

I have created a feature request: https://github.com/Budibase/budibase/issues/5100
